src/Assert.php

```diff
@@ -813,6 +832,42 @@ public static function count($array, $number, $message = '')
         );
     }
 
+    public static function minCount($array, $min, $message = '')
+    {
+        if (count($array) < $min) {
```

Comment:
Okey, fair. I agree with consistency. 
A separate issue should discuss if minCount($x, 3) should make sure that $x is countable..